crates/sui-framework/packages/move-stdlib/sources/type_name.move

```diff
@@ -37,18 +37,15 @@ public struct TypeName has copy, drop, store {
     name: String,
 }
 
-/// Return a value representation of the type `T`.  Package IDs
-/// that appear in fully qualified type names in the output from
-/// this function are defining IDs (the ID of the package in
-/// storage that first introduced the type).
-public native fun get<T>(): TypeName;
-
-/// Return a value representation of the type `T`.  Package IDs
-/// that appear in fully qualified type names in the output from
-/// this function are original IDs (the ID of the first version of
-/// the package, even if the type in question was introduced in a
-/// later upgrade).
-public native fun get_with_original_ids<T>(): TypeName;
+/// Return a value representation of the type `T`. Package IDs that appear in fully qualified type
+/// names in the output from this function are defining IDs (the ID of the package in storage that
+/// first introduced the type).
+public native fun with_defining_ids<T>(): TypeName;
```

Comment:
It might be nice to give an example/explanation for _why_ this and the other are plural, even with a simple tuple example.